### PR TITLE
Finished chapter of DjangoGirls extensions about comment feature

### PR DESCRIPTION
**Pull request by nnkps**
Wrote chapter about creating comment model. Please check for typos and other things that should be corrected :-)

### CODE REVIEW

File: homework_create_more_models/README.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+## Creating comment blog model
+
+Let's open `blog/models.py` and append this piece of code to the end of file:
+
+    class Comment(models.Model):
+    	post = models.ForeignKey('blog.Post', related_name='comments')
+    	author = models.CharField(max_length=200)
+    	text = models.TextField()
+    	created_date = models.DateTimeField(default=timezone.now)
```
Comment (aniav) [inline homework_create_more_models/README.md]: I wonder if `auto_now_add=True` would fit better here and would be simpler to explain. How do you think?

File: homework_create_more_models/README.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+    	approved = models.BooleanField(default=False)
+
+    	def approve(self):
+    		self.approved = True
```
Comment (aniav) [inline homework_create_more_models/README.md]: There are different indentations here and above. I know that this is being picky, but cleaning this up shouldn't be a lot of work :)

File: homework_create_more_models/README.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+If you type `python manage.py runserver` in command prompt and go to [http://127.0.0.1:8000/admin/](http://127.0.0.1:8000/admin/) in your browser, you should have access to list, add and remove comments. Don't hesitate to play with it!
+
+## Make our comments visible
+
+Go to `blog/templated/blog/post_detail.html` file and add those lines before `{% endblock %}` tag:
+
+	<hr>
+	{% for comment in post.comments.all %}
+		<div class="comment">
+			<div class="date">{{ comment.created_date }}</div>
```
Comment (aniav) [inline homework_create_more_models/README.md]: Again different indentations in HTML. Here there are tabs, and below there are spaces :v:

File: homework_create_more_models/README.md [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+			<strong>{{ comment.author }}</strong>
+			<p>{{ comment.text|linebreaks }}</p>
+		</div>
+	{% empty %}
+		<p>No comments here yet :(</p>
+	{% endfor %}
+
+Now we can see the comments section on pages with post details.
+
+But it can look a little bit better, add those some css to `static/css/blog.css`:
```
Comment (aniav) [inline homework_create_more_models/README.md]: Remove "those" (I will add some crazy emoji, so my comments won't look so offensive) :elephant: :tiger:

File: homework_create_more_models/README.md [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+## Let your readers write comments
+
+Right now we can see comments on our blog, but we cannot add them, let's change that!
+
+Go to `blog/forms.py` and add those lines to the end of the file:
+
+    class CommentForm(forms.ModelForm):
+
+    	class Meta:
+    		model = Comment
```
Comment (aniav) [inline homework_create_more_models/README.md]: Indentation again, just check all the code blocks, I won't repeat the comment about indentation any more, you know what's it all about :) :girl: :sheep: :steam_locomotive:

**Comment by aniav** [comment]
Whoweee! This was a lot of work and it's great :sparkles: :+1:

File: homework_create_more_models/README.md (hunk 1 above)
Comment (nnkps) [inline homework_create_more_models/README.md]: Hmm, you're right, but timezone.now is used in django girls tutorial here http://tutorial.djangogirls.org/en/django_models/README.html#creating-a-blog-post-model 
I didn't want to introduce misunderstanding :-)

File: homework_create_more_models/README.md (hunk 2 above)
Comment (nnkps) [inline homework_create_more_models/README.md]: I copied something from Sublime and then rewrite some things in gitbook and I haven't noticed the different indentations. I will fix that. Thank you! :+1:

File: homework_create_more_models/README.md (hunk 4 above)
Comment (nnkps) [inline homework_create_more_models/README.md]: They're not offensive, I should read this once again :yellow_heart:

**Comment by olasitarska** [comment]
@nnkps it'd be awesome if you could make those corrections @aniav suggested, I can't wait to merge it! To update the pull request, simply push another commit to your branch, it'll magically appear here :)

**Comment by nnkps** [comment]
@aniav @olasitarska sorry for my late update :worried:

**Comment by olasitarska** [comment]
@nnkps No worries at all! :)

**Comment by olasitarska** [comment]
Thanks both Anias! :heart:

**Pull request merged**